bom-alpha/bom-alpha.gradle

```diff
@@ -9,7 +9,7 @@ group = "io.opentelemetry.instrumentation"
 archivesBaseName = "opentelemetry-instrumentation-bom-alpha"
 
 rootProject.subprojects.forEach { subproject ->
-  if (!project.name.startsWith("bom")) {
+  if (!subproject.name.startsWith("bom")) {
```

Comment:
This was a bug the BOM may have been missing some of the top level projects as Gradle evaluates BFS, alphabetical (BOM users generally wouldn't have noticed though I think)